tests/integration/api_v3/http/blocks.test.js

```diff
@@ -0,0 +1,346 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import moment from 'moment';
+
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+const endpoint = `${baseUrlV3}/blocks`;
+
+const {
+	goodRequestSchema,
+	badRequestSchema,
+	notFoundSchema,
+	wrongInputParamSchema,
+	metaSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	blockSchema,
+} = require('../../../schemas/api_v3/block.schema');
+
+describe('Blocks API', () => {
+	let refBlock;
+	beforeAll(async () => {
+		[refBlock] = (await api.get(`${endpoint}?limit=1&offset=5`)).data;
+	});
+
+	describe('GET /blocks', () => {
+		it('returns list of blocks when called with no params', async () => {
+			const response = await api.get(`${endpoint}?limit=1`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.length).toEqual(1);
+			response.data.forEach(block => expect(block).toMap(blockSchema));
+			expect(response.meta).toMap(metaSchema);
+		});
```

Comment:
The test case doesn't match the test description. Also, please add checks in all the test cases to ascertain that the default sorting order is followed, unless explicitly specified.

Please add another test case with `limit` specified.

Please ensure that the corresponding changes are reflected in the RPC-based tests as well.